workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -0,0 +1,133 @@
+            var projectPath = Path.GetFullPath("../../");
+
+            var Packages = new List<Package>
+            {
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-dynamic-x86_64-msvc_mt-win32",
+                    TargetPath = $"{nativeDependenciesPath}/Windows/x86_64",
+                    CleanPaths = new List<string> {"include", "worker.lib"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-dynamic-x86_64-gcc_libstdcpp-linux",
+                    TargetPath = $"{nativeDependenciesPath}/Linux/x86_64",
+                    CleanPaths = new List<string> {"include"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-bundle-x86_64-clang_libcpp-macos",
+                    TargetPath = $"{nativeDependenciesPath}/OSX",
+                    CleanPaths = new List<string> {"include"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "csharp_core",
+                    TargetPath = $"{managedDependenciesPath}"
+                },
+                new Package
+                {
+                    Type = "schema",
+                    Name = "standard_library",
+                    TargetPath = $"{projectPath}/build/dependencies/schema/standard_library"
+                },
+                new Package
+                {
+                    Type = "tools",
+                    Name = "schema_compiler-x86_64-win32",
+                    TargetPath = $"{tempPath}/schema_compiler",
+                    Platform = OSPlatform.Windows
+                },
+                new Package
+                {
+                    Type = "tools",
+                    Name = "schema_compiler-x86_64-macos",
+                    TargetPath = $"{tempPath}/schema_compiler",
+                    Platform = OSPlatform.OSX
+                }
+            };
+
+            try
+            {
+                if (!Directory.Exists(tempPath)) Directory.CreateDirectory(tempPath);
+
+                foreach (var package in Packages.Where(p =>
+                    !p.Platform.HasValue || RuntimeInformation.IsOSPlatform(p.Platform.Value)))
+                {
+                    var tempFilePath = Path.Combine(tempPath, $"{package.Name}.zip");
+                    var newVersion = !File.Exists(tempFilePath);
+                    if (newVersion)
+                    {
+                        Console.Out.WriteLine($"Downloading {package.Name} to {tempFilePath}...");
+
+                        Common.RunRedirected("spatial",
+                            new[] {"package", "retrieve", package.Type, package.Name, coreSdkVersion, tempFilePath});
+                    }
```

Comment:
I think this will break if you change SDK versions and the tempFile still exists.

If you make:

`var tempFilePath = Path.Combine(tempPath, $"{package.Name}-{coreSdkVersion}.zip");`

That should avoid this problem

Reply:
`tempPath` is already isolated by version:

`var tempPath = Path.GetFullPath($"./build/CoreSdk/{coreSdkVersion}");`

Reply:
Ah woops - missed that